components/chainhook-cli/src/config/file.rs

```diff
@@ -43,11 +45,36 @@ pub struct LimitsConfigFile {
 
 #[derive(Deserialize, Debug, Clone)]
 pub struct NetworkConfigFile {
-    pub mode: String,
+    pub mode: NetworkConfigMode,
```

Comment:
This fixes a bug when you run `chainhook config new --devnet`, then try to use that config, Chainhook would panic.